src/styles/stars.css

```diff
@@ -0,0 +1,16 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
```

Comment:
Ensure that the `star.svg` file exists in the `src/images` directory. This file is necessary for the default star background image.